features/step_definitions/individual_steps.rb

```diff
@@ -154,21 +154,22 @@ class VlpDocument
   find(IvlPersonalInformation.naturalized_citizen_no_radiobtn).click
   find(IvlPersonalInformation.american_or_alaskan_native_no_radiobtn).click
   find(IvlPersonalInformation.incarcerated_no_radiobtn).click
-  find(IvlPersonalInformation.tobacco_user_yes_radiobtn).click if tobacco_user_field_enabled?
+  #find(IvlPersonalInformation.tobacco_user_yes_radiobtn).click if tobacco_user_field_enabled?
```

Comment:
is this fine to be left out for both DC and ME?

Reply:
Confirmed, this is good to remove